site/source/user-manual/flashing.rst

```diff
@@ -0,0 +1,27 @@
+Getting EmbassyOS
+-----------------
+
+After building your device, you need a copy of EmbassyOS.
+
+* Visit `images.start9.com <https://images.start9.com>`_ and enter the product key ``xxxxxxxx`` (all lower case) to download an image of EmbassyOS.
+
```

Comment:
Instruction here should be to get the image from the latest github release - image server will be deprecated soon